style.css

```diff
@@ -497,6 +499,117 @@ textarea::placeholder {
   margin-top: 55px;
 }
 
+/* popup mobile begins */
+
+.overlays {
+  position: absolute;
+  z-index: 3;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  height: 823px;
+  background: #c1c7d0;
+  visibility: hidden;
+
+  /* display: none; */
+}
+
+.overlays:target {
+  visibility: visible;
+  opacity: 1;
+}
+
+.modals {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  position: absolute;
+  height: 713px;
+  left: calc(50% - 343px / 2);
+  right: calc(50% - 343px / 2);
+  top: 60px;
+  z-index: 2;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+
+  /* visibility: hidden; */
+}
+
+.title {
+  width: 259px;
+  height: 36px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-left: -40px;
+}
+
+.mobile-popup-window .close-btn {
+  position: absolute;
+  color: #67798e;
+  margin-top: 32px;
+  margin-right: 15px;
+  display: flex;
+  align-self: flex-end;
+  align-content: flex-end;
+  font-size: 30px;
+}
+
+.close-btn:target {
+  visibility: visible;
+}
+
+.modals .highlight-text {
+  margin-left: -90px;
+}
+
+.modals .skills {
+  margin-left: -115px;
+  margin-top: -20px;
+}
+
+.border {
+  border: 1px solid #ebecf0;
+  width: 100%;
+  margin-top: -10px;
+}
+
+.modal-links {
+  display: flex;
+  margin-top: 6px;
+  margin-left: -15px;
+}
+
+.see-link {
+  margin-right: 14px;
+}
+
+.modal-links a {
+  margin: 0 0.5rem;
+  font-weight: 700;
+  font-size: 14px;
+  color: #6070ff;
+  border-radius: 6px;
+  background: #fff;
+  border: 1px solid #7f8cff;
+  display: flex;
+  padding: 10px 23px;
+  align-items: center;
+}
+
+/* .desktop-popup-window {
+  display: none;
+} */
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.